lib/src/sql/statements/select.rs

```diff
@@ -68,22 +68,54 @@ impl SelectStatement {
 		let opt = &opt.new_with_futures(false).with_projections(true);
 		// Get a query planner
 		let mut planner = QueryPlanner::new(opt, &self.with, &self.cond);
+		// Used for ONLY: is the limit 1?
+		let limit_is_one_or_zero = self.limit.clone().is_some_and(|l| match l.0 {
+			Value::Number(l) => l.to_usize() <= 1,
+			_ => false,
+		});
```

Comment:
We shouldn't need to use `clone()` here I don't think?

Reply:
Unless you were intending for the user to *have* to specify a `LIMIT` with a number, in order to make it deterministic?